deep-learning/_posts/2018-01-27-CRNN-hyperspectral-classification.md

```diff
@@ -0,0 +1,57 @@
+layout: review
+title:  "Convolutional Recurrent Neural Networks for Hyperspectral Data Classification"
+tags:   hyperspectral, CNN, RNN, classification, deep-learning
+author: Charles Authier
+pdf:    http://www.mdpi.com:8080/2072-4292/9/3/298/pdf
+cite:
+  authors: "Hao Wu, Saurabh Prasad"
+  title:   "Convolutional Recurrent Neural Networks for Hyperspectral Data Classification"
+  venue:   "Remote Sens. 2017"
+---
+
+### Description
+Hyperspectral data, which capture spectral information of objects over the RGB, have played a key role in remote sensing data analysis. With the modern advanced hyperspectral sensors are able to collect hyperspectral images of very high resolution, from which we can collect a large number of labeled pixels for training deep neural networks.
+
+One-dimensional CNNs are already used to extract spectral features for hyperspectral data in some papers. Two-dimensional CNNs had been employed to extract features for hyperspectral data in other.CNNs can also be used for scene classification of high-resolution remote sensing RGB images.
+
+In a paper of 2016, an RNN was used for land cover change detection on multi-spectral images. Normally RNN and LSTM networks have been used for time series data analysis, such as speech recognition, machine translation, etc. But the LSTM network is a special type of RNN, which is able to capture very long-term dependencies embedded in sequence data like the spectral information in hyperspectral data.
+
+So the idea in this paper is to combine the better of the RNN with a CNN. To be able to make a better spatial classification with all the spectral information indisposition. The CRNN is a hybrid of convolutional and recurrent neural networks. It is composed of several convolutional (and pooling) layers followed by a few recurrent layers (Figure 5). First, the convolutional layers are able to extract middle-level, abstract and locally invariant features from the input sequence. The pooling layers help reduce computation and control overfitting. After the recurrent layers extract contextual information from the feature sequence generated by the previous convolutional layers. Contextual information captures the dependencies between different bands in the hyperspectral sequence.
+
+<img src="/deep-learning/images/hyp_crnn/cnn_hyp.png" width="300"><img src="/deep-learning/images/hyp_crnn/rnn_hyp.png" width="400">
+
+For the CNN input, they give the spectral information, which passes in 1D convolutions and max pooling and the result of those convolutional layers as the feature sequence for the RNN input.
+
+<img src="/deep-learning/images/hyp_crnn/crnn_hyp.png" width="700">
+
+To help in the classification, because the neural networks will conserve the spectral features for each pixel independently, the authors add a linear opinion pools (LOP) in the system. Based on spatially neighboring pixels tend to have similar categories. In smoothing out noisy predictions due to noisy data or outliers.
```

Comment:
To improve the pixel classification operation,

Reply:
conserve? => preserve, keep, consider

Reply:
Not sure what you mean by "Based on spatially neighboring pixels tend to have similar categories. In smoothing out noisy predictions due to noisy data or outliers."  Please reformulate

Reply:
What operation is done by the Linear Opinion Pools? Is there an equation in the paper?